spellbook/tests/tornado_cash/tornado_cash_deposits_assert_unique.sql

```diff
@@ -0,0 +1,9 @@
+-- Check that all deposit txs are unique
+WITH unit_tests as
+(SELECT case when COUNT(*) > 1 then True else False end as price_test
```

Comment:
What's the rationale behind a custom test rather than defining a unique test in the schema?

Reply:
None, I thought that checking to make sure that deposits/withdrawals aren't counted twice would make sense, are custom tests not good practice?